.travis.yml

```diff
@@ -164,6 +164,10 @@ matrix:
       script:
         - $MVN test -pl 'web-console'
 
+    - name: "docs"
+      install: cd website && npm install
+      script: cd website && npm run lint
```

Comment:
From the travis log, it looks like the `cd website` done in the `install` step stays in effect when the `script step`.  Some alternatives are to use pushd/popd or a subshell.